tests/System.IO.Pipelines.Performance.Tests/columns/RpsColumn.cs

```diff
@@ -21,10 +21,22 @@ public string GetValue(Summary summary, Benchmark benchmark)
 
         public bool IsDefault(Summary summary, Benchmark benchmark) => false;
         public bool IsAvailable(Summary summary) => true;
+
+        public string GetValue(Summary summary, Benchmark benchmark, ISummaryStyle style)
```

Comment:
I believe he is suggesting to remove RpsColumn completely and do [this](https://github.com/ahsonkhan/corefxlab/blob/cbb539ce791872e63f3af1a2e2f916078e41eb7d/tests/System.IO.Pipelines.Performance.Tests/configs/CoreConfig.cs#L16).